docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.7.1-alpine
+
+LABEL maintainer="Sergey Fedorov <oni.strech@gmail.com>"
+LABEL repository="strech/sidekiq-prometheus-exporter"
+
+ENV RACK_VERSION 2.0.9
+ENV SIDEKIQ_VERSION 5.2.8
+ENV REDIS_NAMESPACE_VERSION 1.7.0
+ENV SIDEKIQ_PROMETHEUS_EXPORTER_VERSION 0.1.13
+
+RUN    gem install "rack:$RACK_VERSION" \
+    && gem install "sidekiq:$SIDEKIQ_VERSION" \
+    && gem install "redis-namespace:$REDIS_NAMESPACE_VERSION" \
+    && gem install "sidekiq-prometheus-exporter:$SIDEKIQ_PROMETHEUS_EXPORTER_VERSION"
+
+WORKDIR /app
+COPY config.ru config.ru
+
+EXPOSE 9292
```

Comment:
run as non-root user?

Reply:
make sense, will switch to non-root user

Reply:
@mikhailadvani Thanks for the advice, non-root user added above ⬆️